### ISSUE TITLE
Bug Report: CyberChef won't run on Firefox 62.0

### ISSUE THREAD

**Issue by loderunner** [gchq/CyberChef#399]
### Summary

An error occured while loading CyberChef

```
TypeError: e is undefined
Filename: blob:https://gchq.github.io/a6cdc597-1800-c443-a173-2b8ca14e1c8c
Line: 1828
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:62.0) Gecko/20100101 Firefox/62.0
CyberChef version: 8.8.5s
```

### Example

On Firefox 62.0, loading CyberChef does not finish.

**Comment by JohnNickerson** [comment]
Same thing for me today in Firefox 63:

TypeError: e is undefined
Filename: blob:https://gchq.github.io/84e1d49c-c5be-47b7-af68-26213b7f5f06
Line: 1828
User-Agent: Mozilla/5.0 (Windows NT 10.0; WOW64; rv:63.0) Gecko/20100101 Firefox/63.0
CyberChef version: 8.8.5s

**Comment by xambroz** [comment]
to me it works online on https://gchq.github.io/CyberChef/index.html,
but when I download it for offline use - the https://gchq.github.io/CyberChef/cyberchef.htm , then it fails with the mentioned error.
Actually it even fails accessing directly the URL https://gchq.github.io/CyberChef/cyberchef.htm.

**Comment by n1474335** [comment]
Fixed in #403. Thanks to @bwhitn.
